EventListener/JmsSerializeListenerAbstract.php

```diff
@@ -124,6 +128,10 @@ protected function serializeValue(LiipImagineSerializableField $liipAnnotation,
         if (count($result) !== 0) {
             $result[$filters] = $filtered;
 
+            if (array_key_exists('includeHost', $this->config) && !$this->config['includeHost']) {
+                $result[$filters] = $this->stripHostFromUrl($result[$filters]);
+            }
+
```

Comment:
Please, put this "if" and "if" in line 118 in private method. (This will remove duplication issue)